packages/now-cli/src/commands/dev/dev.ts

```diff
@@ -1,4 +1,5 @@
 import path from 'path';
+import 'dotenv/config';
```

Comment:
Shouldn’t this line break users projects that aren’t having `dotenv`? I mean, this package isn’t listed in dependencies for the cli, it only listed as `devDependency`. If we fix that, I would say LGTM.